webview_panels/src/modules/queryPanel/components/perspective/PerspectivePlugins.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable */
```

Comment:
We should fix the TS errors and make this TS file. Keeping it as JS will make the code non typesafe and difficult to make changes in future

Reply:
The code for the plugin is a bit involved types-wise. So, Aditya and I decided to use Js as it works correctly. [2b47fab](https://github.com/AltimateAI/vscode-dbt-power-user/pull/1354/commits/2b47fab24d3456f3e2664ba2c179ae2d08168092) has the original file with types if you want to take a shot at it.